cdk/resources/bootstrap.sh

```diff
@@ -215,6 +215,44 @@ git checkout $WORKSHOP_GIT_BRANCH
 
 cp hack/.zshrc hack/.p10k.zsh ~/
 
+# Copy VPC folder and start VPC creation in background
+cp -r $BASE_DIR/terraform/vpc /home/ec2-user/environment/
+cd /home/ec2-user/environment/vpc
+nohup bash -c "terraform init && terraform apply -auto-approve && echo 'VPC_CREATION_COMPLETE' > /tmp/vpc_status.flag" > /tmp/vpc_creation.log 2>&1 &
```

Comment:
I am not sure this is solid enough to scale. In fleet we are using codebuild to create stacks in a more robust fashion